src/components/learner-credit-management/SubBudgetCard.jsx

```diff
@@ -49,24 +49,26 @@ const SubBudgetCard = ({
 
   const renderCardHeader = (budgetType, budgetId) => {
     const subtitle = (
-      <Stack direction="horizontal" gap={2.5}>
+      <Stack direction="horizontal" gap={2.5} className="pb-4">
         <Badge variant={budgetLabel.badgeVariant}>{budgetLabel.status}</Badge>
         <span data-testid="budget-date">
           {budgetLabel.term} {formattedDate}
         </span>
       </Stack>
     );
-
+    const action = budgetLabel.status !== BUDGET_STATUSES.scheduled
+      ? renderActions(budgetId)
+      : undefined;
     return (
-      <Card.Header
-        title={<BackgroundFetchingWrapper>{budgetType}</BackgroundFetchingWrapper>}
-        subtitle={<BackgroundFetchingWrapper>{subtitle}</BackgroundFetchingWrapper>}
-        actions={
-          budgetLabel.status !== BUDGET_STATUSES.scheduled
-            ? renderActions(budgetId)
-            : undefined
-        }
```

Comment:
I would suggest leaving the CTA within the `actions` prop of `Card.Header` (to take advantage of default styling / sizes of the action CTAs for `Card`). In order to get the action to be vertically centered within the `Card.Header` (instead of the default positioning), you could likely pass the `align-items-center` class name to the `Card.Header` component.

```jsx
<Card.Header
  className={classNames(
    'align-items-center',
    {
      'mb-4.5': budgetLabel.status !== BUDGET_STATUSES.active,
    }
  )}
  // ...
>
```